CONTRIBUTING.md

```diff
@@ -151,7 +152,11 @@ git fetch --all --prune
 ---
 
 #### **New Feature**
+SST is an open source project and we encourage contribution of new features. For new features, please open a “Feature Request” in the GitHub issues tracker and describe both the use-case and the proposed functionality in sufficient detail that the community can understand how the feature will generally integrate into the existing code base.  Once the code is ready, a GitHub pull request (PR) should be opened as described in this document.  If the feature can be implemented using one of the existing “plug-in” APIs that already exist in the core, then it is preferred that the development use these APIs.  Features implemented through plug-ins can be made available externally and do not necessarily need to be merged into the main code repository, though that option is available if there is a compelling reason.
```

Comment:
"If the feature can be implemented using one of the existing "plug-in" APIs in the core, ..." (currently says "existing" twice)